cluster/images/etcd/migrate/migrate_server.go

```diff
@@ -105,7 +105,7 @@ func (r *EtcdMigrateServer) Stop() error {
 	}
 	gracefulWait := time.Minute * 2
 	stopped := make(chan bool)
-	timedout := make(chan bool)
+	timedout := make(chan bool, 1)
```

Comment:
Instead of that, instead of sending an item to a channel, we should simply close it.

Reply:
fixed